BOOTSEQ.md

```diff
@@ -4,77 +4,160 @@
+  - Provide power to xPU without starting host
+  - Wait for xPU to complete startup
+  - Boot host
+
+- Who tells host OS to continue / start booting ?
+  - Default is for host OS continues boot process unless it is paused by CRS signal from xPU ?
+
+- Are there any circumstances under which host CPU is halted ?
+
+- Can we wait in UEFI/BIOS in case of network boot via xPU ?
   - Some UEFI PXE/HTTP boots have timeouts, sometimes painfully short ones (as low as 30 seconds)
-  - If the XPU provides an UNDI driver for network boot in its OPROM, it can have the Host BIOS wait.
+  - If the xPU provides an UNDI driver for network boot in its OPROM, it can have the host BIOS wait.
+
 - Should host continue booting even if xPU failed to boot after timeout ?
-  - Yes. XPU services/devices will be unavailable
+  - Yes. xPU services/devices will be unavailable
+  - Yes. May be needed to overwrite a corrupt image on the xPU
+
+- Should xPU OS wait for host to finish boot ?
+  - Only if use case requires the xPU to have dependencies on resources owned by the host ?
+
+- More...
+
+## Host Bootup/Restart
+
+1. xPU already executing in always-on-power slot
+2. Host reboot; xPU ignores reboot and continues executing during host reboot
+
+Use cases
+
+1. Host is providing power and cooling to xPU; there is no dependency by the xPU on any host functionality
+
+Events:-
+
+- Host is started/restarted
+- All resources provided by the xPU are already online and can be used by host during boot phase
+- Host can use the xPU for network boot without timeout issues
+- More...
+
+## xPU Bootup/Restart
+
+1. xPU restarting after reset or crash; DPC event did not result in host restart or crash
+2. xPU powered on using host BIOS to provide power to PCIe slot without starting host
+
+- xPU starts booting
+- xPU completes boot and issues hot-plug event
+- Host updates root complex with xPU resources and makes those resources available to services executing on the host
+- _Note: This assumes host will properly execute a hot-plug event_
 - More...
 
-## xPU and host reboot
+## Host OS Reset or Crash
+
+- During host OS Reboot or Crashes, the host writes to Chipset Reset Control Register (i.e., IO port 0xCF9) or to the ACPI FADT
+- The Platform (CPLD) firmware can monitor for these events and trigger host BMC/BIOS to coordinate with xPU to take appropriate action. Potential actions include:-
+  - Ensuring that xPU is gracefully shut down before a full host reboot
+  - Force reset xPU after a timeout if xPU has failed to gracefully shutdown
+  - Allow xPU to continue executing if there is no host/xPU dependency
+
+## xPU OS Reset or Crash
+
+xPU OS/FW crash and/or independent reboot will result in a Downstream Port Containment (DPC) event on the PCIe bus
+
+- The host (BMC/BIOS) firmware and host OS needs to implement DPC and hot plug requirements defined in PCIe Firmware spec 3.3 for these scenarios to work.
```

Comment:
Ack. This also requires some basic mechanism for BMC/BIOS to learn that a xPU exist in the enumeration of devices on the server.